api/src/main/scala/com/azavea/franklin/api/Server.scala

```diff
@@ -0,0 +1,110 @@
+package com.azavea.franklin.api
+
+import doobie.hikari.HikariTransactor
+import doobie.util.ExecutionContexts
+import org.http4s.implicits._
+import org.http4s.server.blaze._
+import org.http4s.server.middleware._
+import org.http4s.server.{Router, Server => HTTP4sServer}
+import tapir.docs.openapi._
+import tapir.openapi.circe.yaml._
+import tapir.swagger.http4s.SwaggerHttp4s
+import cats.effect._
+import com.azavea.franklin.api.commands.{ApiConfig, Commands, DatabaseConfig}
+import com.azavea.franklin.endpoints.{
+  CollectionEndpoints,
+  CollectionItemEndpoints,
+  LandingPageEndpoints,
+  SearchEndpoints
+}
+import com.azavea.franklin.services.{
+  CollectionItemsService,
+  CollectionsService,
+  LandingPageService,
+  SearchService
+}
+import cats.implicits._
+
+object Server extends IOApp {
+
+  private val banner: List[String] =
+    """
+   $$$$$$$$$
+$$$$$$$$$$$$   ________                            __        __  __
+$$$$$$$$$$$$  /        |                          /  |      /  |/  |
+$$            $$$$$$$$/______   ______   _______  $$ |   __ $$ |$$/  _______
+ $$$$$$$$     $$ |__  /      \ /      \ /       \ $$ |  /  |$$ |/  |/       \
+$$$$$$$$      $$    |/$$$$$$  |$$$$$$  |$$$$$$$  |$$ |_/$$/ $$ |$$ |$$$$$$$  |
+$$$$$$$       $$$$$/ $$ |  $$/ /    $$ |$$ |  $$ |$$   $$<  $$ |$$ |$$ |  $$ |
+$             $$ |   $$ |     /$$$$$$$ |$$ |  $$ |$$$$$$  \ $$ |$$ |$$ |  $$ |
+$$$$$$        $$ |   $$ |     $$    $$ |$$ |  $$ |$$ | $$  |$$ |$$ |$$ |  $$ |
+$$$$$         $$/    $$/       $$$$$$$/ $$/   $$/ $$/   $$/ $$/ $$/ $$/   $$/
+$$$$
+ 
+""".split("\n").toList
+
+  private def createServer(
+      apiConfig: ApiConfig,
+      dbConfig: DatabaseConfig
+  ): Resource[IO, HTTP4sServer[IO]] =
+    for {
+      connectionEc  <- ExecutionContexts.fixedThreadPool[IO](2)
+      transactionEc <- ExecutionContexts.cachedThreadPool[IO]
+      xa <- HikariTransactor.newHikariTransactor[IO](
+        "org.postgresql.Driver",
+        dbConfig.jdbcUrl,
+        dbConfig.dbUser,
+        dbConfig.dbPass,
+        connectionEc,
+        transactionEc
+      )
+      allEndpoints      = LandingPageEndpoints.endpoints ++ CollectionEndpoints.endpoints ++ CollectionItemEndpoints.endpoints ++ SearchEndpoints.endpoints
+      docs              = allEndpoints.toOpenAPI("Franklin", "0.0.1")
+      docRoutes         = new SwaggerHttp4s(docs.toYaml, "open-api", "spec.yaml").routes[IO]
+      landingPageRoutes = new LandingPageService[IO].routes
+      searchRoutes      = new SearchService[IO](apiConfig, xa).routes
+      collectionRoutes = new CollectionsService[IO](xa).routes <+> new CollectionItemsService[IO](
+        xa
+      ).routes
+      router = CORS(
+        Router(
+          "/" -> ResponseLogger.httpRoutes(false, false)(
```

Comment:
Since we're not putting this behind a reverse proxy I thought it would be nice to have some request/response logging -- at some point we will probably want to substitute this out for our own that gives us more information akin to nginx access logs.